frontend-client/public/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <!--Font Awesome Import-->
+    <script src="https://kit.fontawesome.com/b9f3787362.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab&display=swap" rel="stylesheet"> 
```

Comment:
I'm not sure if for this project an external online dependency (Google Fonts and Font Awesome) is ideal. A local version of the font might be the better way to go in this situation, if there might be the chance that the refbox is not connected to the internet (or has a slow/unstable connection)